README.md

```diff
@@ -1,3 +1,17 @@
+# Note: forked
+
+This resource has been forked to add support for [retrieving AWS credentials
+from IAM instance profiles][IAM]. [Docker Hub][hub] builds a container from
+the `gds` branch. There is no automated testing because the integration
+tests require credentials, so you will need to run the tests yourself.
+
```

Comment:
Maybe it would add more context if you add a reference to the discussion in https://github.com/concourse/s3-resource/pull/22 and mention that the concourse developers and looking at alternative long term solutions to the problem of retrieving credentials.